project/Docs.scala

```diff
@@ -47,8 +33,6 @@ object Docs {
       "gray-lighter" -> "#F4F3F4",
       "white-color" -> "#FFFFFF"),
     autoAPIMappings := true,
-    unidocProjectFilter in(ScalaUnidoc, unidoc) :=
```

Comment:
@kailuowang Removing this line seems to be generating compilation errors during the build:

```
info] Main Scala API documentation to /home/travis/build/typelevel/cats-mtl/docs/target/scala-2.11/unidoc...
[error] /home/travis/build/typelevel/cats-mtl/core/src/main/scala/cats/mtl/ApplicativeAsk.scala:24: ApplicativeAsk is already defined as trait ApplicativeAsk
[error] trait ApplicativeAsk[F[_], E] extends Serializable {
[error]       ^
[error] /home/travis/build/typelevel/cats-mtl/core/src/main/scala/cats/mtl/ApplicativeAsk.scala:32: ApplicativeAsk is already defined as object ApplicativeAsk
```
Full log: https://travis-ci.org/typelevel/cats-mtl/jobs/426334463

Reply:
try set it to `Seq(coreJVM, lawsJVM)`?

Reply:
The previous comment seemed to be saying ghaty an empty sequence is equivalent to setting all the projects: kernelJVM, coreJVM, freeJVM

Reply:
Those are copied from Cats. 
I'd say let's copy from the settings in cats now. 
https://github.com/typelevel/cats/blob/master/build.sbt#L198
in our case it's just coreJVM and lawsJVM

Reply:
Ok, great. Will do.

Reply:
@kailuowang So all example settings I've seen (i.e. in cats), the definition for the docs resides inside the `build.sbt`. Specifically here, we're inside another `Docs` object, which has no access to `coreJVM` or `lawsJVM` values defined.

Should I move this setting into `build.sbt` directly? Or perhaps I'm missing something here.

Reply:
oh I didn't realize that they are in a different file.  I would propose that we move this back to build.sbt.  I am more in the everything in `build.sbt` camp.

Reply:
+1 to "everything in `build.sbt`"